src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/tests/latest/mydomain.test.encrypted.pem

```diff
@@ -49,3 +49,5 @@ OqewlEWAK6Jc6ZPSGzkjTrntGKDoiBtH2g9+VrD8hibqC2K+4cyrkt+SGubh3jtW
 1emPqajxmcw99SYmyZcjZZ25K6x/zaR905ACAzmFwsrnBjSwAHXzsWWZP8Y6nRjs
 YmDigry7A/JhVfckoS1jw5YFjKkpZyzI97YXRSAaWr5F9jqFO8s=
 -----END CERTIFICATE-----
+
+openssl genrsa 2048 | openssl pkcs8 topk8 -v2
```

Comment:
Is this allowed in a certificate file?

Reply:
No, I but this cert isn't used in tests anymore I will remove it